### PR TITLE
migrations exercises

### PR DESCRIPTION
**Pull request by Ankita9818**

### CODE REVIEW

File: db/migrate/20180413101514_change_default_enabled_of_products.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeDefaultEnabledOfProducts < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default :products, :enabled, from: true, to: false
```
Comment (nimish13) [inline db/migrate/20180413101514_change_default_enabled_of_products.rb]: why have we used `true` in `from` option?
Shouldn't it be nil?

File: db/migrate/20180413111835_irreversible_migration_example.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class IrreversibleMigrationExample < ActiveRecord::Migration[5.1]
+  def up
```
Comment (nimish13) [inline db/migrate/20180413111835_irreversible_migration_example.rb:2]: not a proper example of irreversible migration. Read more about the same

File: db/migrate/20180413101311_add_details_to_products.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class AddDetailsToProducts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :products, :enabled, :boolean
+    add_column :products, :discount_price, :decimal, precision: 5, scale: 2
```
Comment (nimish13) [inline db/migrate/20180413101311_add_details_to_products.rb]: why precision is just 5. Any specific reason?

File: app/views/users/_form.html.erb [hunk 4]
```diff
@@ -22,6 +22,12 @@
     <%= f.text_field :name, size: 40 %>
   </div>
 
+<!-- *** migration exercise no 6 ***  -->
+  <div class="field">
+    <%= f.label :name, 'Email:' %>
```
Comment (nimish13) [inline app/views/users/_form.html.erb]: why have we passed `:name` to label?

Comment (Ankita9818) [reply app/views/users/_form.html.erb]: corrected it

File: db/migrate/20180413111835_irreversible_migration_example.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+  # def down
+  #   # It raises ActiveRecord::IrreversibleMigration exception
+  #   # To roll back this migration , we would have to add it
+  #   add_column :users, :email, :string
+  # end
+  def change
+    reversible do |dir|
+      change_table :products do |t|
+        dir.up   { t.change :price, :decimal, :precision => 8, :scale => 3 }
```
Comment (nimish13) [inline db/migrate/20180413111835_irreversible_migration_example.rb]: please don't change the precision of price column. It may be used in other exercises

File: app/views/users/_form.html.erb [hunk 6]
```diff
@@ -25,7 +25,7 @@
 <!-- *** migration exercise no 6 ***  -->
   <div class="field">
     <%= f.label :email, 'Email:' %>
-    <%= f.email_field :email %>
+    <%= f.text_field :email %>
```
Comment (Ankita9818) [inline app/views/users/_form.html.erb]: this is not related to migrations' exercise.
I have changed it for checking html validations.
